scripts/install/sabnzbd.sh

```diff
@@ -10,30 +10,51 @@
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
 
-user=$(cut -d: -f1 < /root/.master.info)
-password=$(cut -d: -f2 < /root/.master.info)
-distribution=$(lsb_release -is)
-codename=$(lsb_release -cs)
-#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
-latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
-latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
-
 . /etc/swizzin/sources/functions/pyenv
 . /etc/swizzin/sources/functions/utils
 
-LIST='par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
+user=$(_get_master_username)
+password=$(_get_user_password ${user})
+#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
+latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | jq -r '.assets[]?.browser_download_url | select(contains("tar.gz"))') || {
+    echo_error "Failed to query GitHub for latest sabnzbd version"
+    exit 1
+}
+latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
```

Comment:
```suggestion
ret=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest) || {
    echo_error "Failed to query GitHub"
    exit 1
}
latest=$(echo $ret | jq -r '.assets[]?.browser_download_url | select(contains("tar.gz"))') 
latestversion=$(echo $ret | jq -r .tag_name)
```
`jq` > `awk` & `cut`ing `JSON`s

Reply:
It's not `awk`'ing json if you closely inspect the output of `latest`. It's parsing the URL itself.

Reply:
Ah yes I didn't look closely enough

Reply:
Furthermore, I believe my current implementation of grabbing the full url in one command is more resilient as your command will only fail if the API is down, but the committed command will fail if a `.tar.gz` link is not returned.

Reply:
Truth!